docs/contributing/governance.rst

```diff
@@ -0,0 +1,134 @@
+   regularly to discuss the technical direction of the project. ATS
+   reviews and approves major changes, proposals, and contributions that
+   could impact the broader user and developer community. See the `ATS
+   approval` process described in :ref:`labelling-guidelines` for
+   details on how technical decisions are flagged and processed by this
+   group.
+
+-  **@anemoisecurity group:** This group is responsible for managing
+   access to sensitive components of the project infrastructure,
+   including GitHub permissions and package publishing. The group also
+   acts as the final gatekeeper for pull request merges that require
+   higher scrutiny or formal review through ATS.
+
+As the project matures, we expect to refine and expand the governance
+model, potentially including additional roles, working groups, and
+decision-making mechanisms to support transparency, scalability and
+collaboration.
+
+*********************************
+ Anemoi Technical Subgroup (ATS)
+*********************************
+
+Anemoi is an open-source machine learning framework co-developed by
+several national meteorological institutions in collaboration with
+ECMWF. Details about individual contributors can be found in the
+`CONTRIBUTORS.md
+<https://github.com/ecmwf/anemoi-docs/blob/main/CONTRIBUTORS.md>`_ file
+in each package. Inspired by the governance structures of other
+open-source projects such as PyTorch and TensorFlow, **Anemoi is guided
+by the Anemoi Technical Subgroup (ATS)**, a committee composed of
+representatives from each contributing institution. The ATS meets weekly
+in what is known as the ATS meeting, which serves as a forum to
+coordinate the project's technical direction. During these meetings, we
+discuss ongoing development to ensure that the codebase evolves in a way
+that meets the needs of all participating organisations. This includes
+reviewing major pull requests (PRs), proposals for new features, and any
+changes that could introduce breaking behavior or support new use cases.
+
+PRs and issues intended for ATS discussion are identified using the
+``ATS Approval Needed`` label. Refer to the :ref:`labelling-guidelines`
+for more information on when and how to apply this label.
+
+Additionally, contributors can tag ``@ATS`` in issues or PRs to
+explicitly highlight topics requiring subgroup attention. This can be
+especially helpful to summarize a disagreement, flag a potentially
+controversial feature, or raise visibility for decisions that require
+alignment across institutions.
+
+***********************
+ Anemoi Security Group
+***********************
+
+The ``@anemoisecurity`` group is a core part of Anemoi’s governance and
+review process. It is based on GitHub’s `CODEOWNERS
+<https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners>`_
+mechanism, and is defined at the organisation level:
+https://github.com/orgs/ecmwf/teams/anemoisecurity.
```

Comment:
Does having this URL add anything? it's private

Reply:
Good point, I hadn't realised that was private. My intention with adding a link was to make more transparent who's the security group. Will remove the link and simply mention, that to reach out or ping the security group you can just simply do '@ecmwf/anemoisecurity'